lib/sda/tree.go

```diff
@@ -403,9 +428,14 @@ type TreeNode struct {
 	Id uuid.UUID
 	// The Entity points to the corresponding host. One given host
 	// can be used more than once in a tree.
-	Entity   *network.Entity
-	Parent   *TreeNode
+	Entity *network.Entity
+	// Parent link
+	Parent *TreeNode
+	// Children links
 	Children []*TreeNode
+	// Aggregate public key for *this* subtree,i.e. this node's public key + the
+	// aggregate of all its children's aggregate public key
+	PublicAggregateSubTree abstract.Point
```

Comment:
Just to be sure: `CoSi` verifies against the aggregated public key in the same way, that is, all children PLUS the current host?